includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -984,7 +984,17 @@ public function get_upe_available_payment_methods() {
 		$methods[] = Sofort_Payment_Method::PAYMENT_METHOD_STRIPE_ID;
 		$methods[] = Sepa_Payment_Method::PAYMENT_METHOD_STRIPE_ID;
 		$methods[] = P24_Payment_Method::PAYMENT_METHOD_STRIPE_ID;
-		$methods[] = Link_Payment_Method::PAYMENT_METHOD_STRIPE_ID;
+
```

Comment:
So it was the capability ignored. Can we slightly refactor this to be more universal?

```php
        $capabilities = $this->get_payment_method_capability_key_map();
		if ( isset( $capabilities[ Link_Payment_Method::PAYMENT_METHOD_STRIPE_ID ] ) )  {
			$payment_methods_statuses = $this->get_upe_enabled_payment_method_statuses();
			$link_status              = $payment_methods_statuses[ $capabilities[ Link_Payment_Method::PAYMENT_METHOD_STRIPE_ID ] ]['status'];
			if ( $link_status === 'active' ) {
				$methods[] = Link_Payment_Method::PAYMENT_METHOD_STRIPE_ID;
			}
		}
```

Reply:
`$link_status === 'active'` - I would keep `$link_status !== 'inactive'`, because I noticed that the status can be also `unrequested` . 
What do you think @kalessil ?

Reply:
In that case, it still not be available for use in payments. That's why I suggested using `$link_status === 'active'`.

Reply:
Update: with new input from Dan we agreed to proceed with `$link_status !== 'inactive'`

Reply:
@dpaun1985 : can you also update PR testing instructions please, to test the US account with the enabled/disabled link capabilities?

Reply:
@frosso, @ismaeldcom: I'm not sure how much you are remembered about this logic, but what we don't rely on account capabilities for building the list initially (and later cross-checking it against available fees)?

Reply:
Sorry @kalessil , I'm not familiar with the reasoning behind it 😬 maybe because the cached account data might not be available when we build the list (and we might want to display the payment methods on the settings page anyways)?

Reply:
AFAIK we don't use account capabilities because this list is used to show available payment methods to be enabled/disabled and their capability requested. If we build it based on current capabilities, they will not see any additional payment method to enable on the list (in settings).

I understand it like a list of available UPE payment methods on our platform and merchant country. Not tied to merchant account capabilities.

Reply:
>I understand it like a list of available UPE payment methods on our platform and merchant country. Not tied to merchant account capabilities.

For some countries, not all payment methods can be enabled. In this case, in Stripe response, that capability will not be present. Based on Stripe response I think we can hide/show available payment methods in the admin -> if a capability is not present ( no matter the status ), that means it is not available for that merchant ( it's not in an eligible country... or any other reason )

Reply:
That's what we're doing with the intersection with fees. It allows us to better control which payment methods to show. Because Stripe might support it but we don't. And as we manually add fees for every country and payment method, that will match our available payment methods 🙂